carto/kuvizs.py

```diff
@@ -42,6 +43,28 @@ class Meta:
         collection_endpoint = API_ENDPOINT.format(api_version=API_VERSION)
         name_field = "id"
 
+    def update(self, html, name, password):
+        self.data = base64.b64encode(html.encode()).decode('ascii')
+        self.name = name
+
+        if password:
+            self.privacy = PRIVACY_PASSWORD
+            self.password = password
+        elif self.password:
+            self.privacy = PRIVACY_PUBLIC
+            self.password = None
+
+        self.save()
+
+    def __repr__(self):
```

Comment:
IMO, `__repr__`s should print code that's reproducible, so something to the effect of:
```python
Kuviz({id}, {url}, {name}, {privacy})
```

I don't see a constructor here, though, so I'm not sure how best to represent an instance of this class.

Reply:
Ok, I will change it for `__str__`

Reply:
Good idea